src/Microsoft.DotNet.XHarness.CLI/Commands/WASM/Browser/WasmTestBrowserCommand.cs

```diff
@@ -175,6 +175,7 @@ protected override async Task<ExitCode> InvokeInternal(ILogger logger)
             {
                 // added based on https://github.com/puppeteer/puppeteer/blob/main/src/node/Launcher.ts#L159-L181
                 "--enable-features=NetworkService,NetworkServiceInProcess",
+                "--allow-insecure-localhost",
```

Comment:
Why is this needed?

Reply:
Because some of the tested connections use HTTPS, but we do not install any trusted certificates. 
Since we do not own TLS layer in the Browser platform, we don't have to test it.
Just keeping it simple.